src/observable/ErrorObservable.ts

```diff
@@ -63,7 +63,7 @@ export class ErrorObservable<T> extends Observable<any> {
     subscriber.error(error);
   }
 
-  constructor(public error: T, private scheduler?: IScheduler) {
+  constructor(public readonly error: any, private readonly scheduler?: IScheduler) {
```

Comment:
yes, we've done similar (https://github.com/ReactiveX/rxjs/pull/2249) and we've accepted to introduce those changes, since it brings explicit clarification for TS users.